pkg_building.Rmd

```diff
@@ -255,11 +255,18 @@ Please do not list editors as contributors. Your participation in and contributi
 
 * Use `Imports` instead of `Depends` for packages providing functions from other packages. Make sure to list packages used for testing (`testthat`), and documentation (`knitr`, `roxygen2`) in your `Suggests` section of package dependencies (if you use `usethis` for adding testing infrastructure via [`usethis::use_testthat()`](https://usethis.r-lib.org/reference/use_testthat.html) or a vignette via [usethis::use_vignette()](https://usethis.r-lib.org/reference/use_vignette.html), the necessary packages will be added to `DESCRIPTION`). If you use any package in the examples or tests of your package, make sure to list it in `Suggests`, if not already listed in `Imports`.
 
+* If your (not Bioconductor) package depends on Bioconductor packages, make sure the installation instructions in the README and vignette are clear enough.
+    
+    * Is the automatic installation of Bioconductor packages by `install.packages()` enough? In that case, mention the user needs to run `setRepositories()` if they haven't set the necessary Bioconductor repositories yet. 
+    
+    * Should the user use [`BiocManager`](https://www.bioconductor.org/install/index.html#why-biocmanagerinstall)? Document this.
+    
+    * If your package depends on Bioconductor after a certain version, mention it in DESCRIPTION and in the installation instructions.
+
```

Comment:
Regarding CI, using `r: bioc-release` is the most appropriate for testing bioconductor packages but it will not flag issues with dependencies that won't install properly from the source code (which is the review scenario). That's why in `BaseSet` review I [suggested an additional build using `r: release`](https://github.com/llrs/BaseSet/blob/master/.travis.yml) be included which can catch such problems. Including any setup instructions in the `script:` filed for that job then also allows testing the suitability of given instructions.  This seemed the most sensible approach to me but happy for feedback on this!

Reply:
BaseSet is a Bioconductor package right? I wonder whether we need different guidance for Bioconductor packages with Bioconductor dependencies vs not Bioconductor packages with Bioconductor dependencies. :thinking:

Reply:
I will not publish BaseSet to Bioconductor I want it on CRAN. All the Bioconductor's (suggested) dependencies are so for historical reasons... I will remove the dependency of the suggested packages on Bioconductor. Part of the "target audience" is there tough.  

I am sure more guidance for packages aiming to be on Bioconductor or depending on Bioconductor's package will be helpful